Cargo.toml

```diff
@@ -48,3 +48,4 @@ reqwest = { version = "0.11", default-features = false, features = ["json"] }
 urlencoding = "2"
 enum-display = "0.1"
 itertools = "0.11"
+prometheus = "0.13"
```

Comment:
Can we make this optional?

Reply:
You mean using featurs?

Reply:
> You mean using featurs?

Yep, databend will not use `prometheus` and may add `prometheus-client` support instead in the future.

Reply:
Done